### PR TITLE
React Capstone project:

### PR DESCRIPTION
**Pull request by algerina**
For this Capstone project, I created a mobile App that checks a list of metrics provided from NovenCovid API.

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @algerina, great job you are doing so far in this project. 💯
TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- Your app is getting the data properly
- Great job getting the data from the API 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️

- [x] Kindly add the link to the live version of your app. 

tempsnip

- [x] Please make sure that there's a link from the Detail page to the Home page by clicking the "Back" button. Currently, the back button is on the home page and it sends me to an empty page. Please fix this. 

- [x] Please make sure you follow the same design from the given design.  Currently, there is only one column, and it must be two columns on the home page. 

Capture

- [x] Please add unit tests for the components of the app and some pure functions in Redux code. Currently, you only have one test and are failing. 

See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

cat


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Card = (params) => {
+  const { country, todayCases, countryInfo: { flag } } = params.data; // eslint-disable-line
+  return (
+    <div className="tc bg--light-blue dib pa1 grow bw2 shadow-5 w-48-ns">
```
Comment (petudeveloper) [inline src/components/Card.js:6]: Your design looks good but you are not following the Nelson Sakwa design. Please try adding the corresponding CSS properties (Font size, spacing, etc) to make your application have a similar style to the given one.

Check the image below for a better understanding:
Capture

**Review by petudeveloper — CHANGES_REQUESTED** [review]
Hi @algerina, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @algerina 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**